src/system/sargon_os/sargon_os_accounts.rs

```diff
@@ -0,0 +1,818 @@
+use std::sync::RwLockWriteGuard;
+
+use crate::prelude::*;
+
+// ==================
+// Create Unsaved Account(s)
+// ==================
+#[uniffi::export]
+impl SargonOS {
```

Comment:
it would not be a field - stored properties does not exist in Rust - but rather a method. and hmm... yeah I need to give it a bit of thought, not so easily done.

Reply:
Hmm quite hard to do, can explain in a meeting later!